articles/tutorials/qft-tutorial.md

```diff
@@ -0,0 +1,624 @@
+print(measurementResult)
+
+# reversing order to show corresponding basis state in binary form
+binaryCompBasisState = ""
+for i in measurementResult:
+    binaryCompBasisState = str(i) + binaryCompBasisState
+print("Corresponding basis state in binary:")
+print("|" + binaryCompBasisState + ">")
+```
+
+Run the file, and your output should look similar to the following:
+
+```Output
+Before measurement: 
+# wave function for qubits with ids (least to most significant): 0;1;2
+|0>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|1>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|2>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|3>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|4>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|5>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|6>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+|7>:	 0.353553 +  0.000000 i	 == 	***                  [ 0.125000 ]     --- [  0.00000 rad ]
+After measurement: 
+# wave function for qubits with ids (least to most significant): 0;1;2
+|0>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|1>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|2>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|3>:	 1.000000 +  0.000000 i	 == 	******************** [ 1.000000 ]     --- [  0.00000 rad ]
+|4>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|5>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|6>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+|7>:	 0.000000 +  0.000000 i	 == 	                     [ 0.000000 ]                   
+
+Post-QFT measurement results [qubit0, qubit1, qubit2]: 
+[1, 1, 0]
+
+Corresponding basis state in binary:
+|011>
+```
+
+#### [C#](#tab/tabid-csharp)
+Now that our operation is returning a result, replace the method call `Wait()` with fetching the `Result` property. 
+This still accomplishes the same synchronicity discussed earlier, and we can directly bind this value to the variable `measurementResult`.
+
+```csharp
+using System;
+
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Quantum.Operations
+{
+    class Driver
+    {
+        static void Main(string[] args)
+        {
+            using (var qsim = new QuantumSimulator())
+            {
+                var measurementResult = Perform3QubitQFT.Run(qsim).Result;
+                System.Console.WriteLine(
+                    $"Post-QFT measurement results [qubit0, qubit1, qubit2]: ");
```

Comment:
Here and on line 541, surely the `$` is redundant?

Reply:
Whoops, good catch!